### PR TITLE
Update files from CI

### PR DESCRIPTION
**Pull request by bmbouter**
This also rewrites the signature tests to use the pytest signing service
fixtures from pulpcore.

[noissue]

### CODE REVIEW

**Comment by dkliban** [comment]
Please update the template_config.yaml with `aiohttp_fixtures_origin: 127.0.0.1` and reapply the plugin_template.

File: pulp_ansible/tests/functional/api/collection/v3/test_deletion.py [hunk 1]
```diff
@@ -201,6 +202,7 @@ def test_delete_deprecated_content(self):
 
         assert len(latest_version.content_summary.present) == 0
 
+    @unittest.skip("needs to use signing fixtures from pulpcore")
```
Comment (bmbouter) [inline pulp_ansible/tests/functional/api/collection/v3/test_deletion.py:205]: @mdellweg and @gerrod3 I don't think we have time to convert this and then release. I'm proposing we accept this skip for this PR, release, and then I'll focus on converting this module to pytest as my next piece of work. I won't get distracted and forget to do it. Is this ok with you? There is one more skip test here too (somewhere else in the PR).

File: pulp_ansible/tests/functional/api/collection/test_signatures.py [hunk 2]
```diff
@@ -1,230 +1,206 @@
...
+def test_collection_signatures_no_deletion(ansible_collection_signatures_client):
+    """Test the API doesn't provide a delete for a Collection Signature obj."""
+    with pytest.raises(AttributeError):
+        ansible_collection_signatures_client.delete
+
+
+# Tests for syncing Collection Signatures
+
+
+def distro_serving_one_signed_one_unsigned_collection(
```
Comment (gerrod3) [inline pulp_ansible/tests/functional/api/collection/test_signatures.py:128]: Think this needs a fixture mark.

Comment (bmbouter) [reply pulp_ansible/tests/functional/api/collection/test_signatures.py:128]: It's a fixture not a test, so I'm not sure I can even mark it?

Comment (gerrod3) [reply pulp_ansible/tests/functional/api/collection/test_signatures.py:128]: Wdym? If it's a pytest fixture doesn't it need the `@pytest.fixture` mark?

**Comment by mdellweg** [comment]
This has been adopted in another PR.

**Pull request closed without merging**